tests/llvm/datasets/anghabench_test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Tests for the AnghaBench dataset."""
+import sys
+from itertools import islice
+from pathlib import Path
+
+import gym
+import pytest
+
+import compiler_gym.envs.llvm  # noqa register environments
+from compiler_gym.envs.llvm import LlvmEnv
+from compiler_gym.envs.llvm.datasets import AnghaBenchDataset
+from tests.pytest_plugins.common import skip_on_ci
+from tests.test_main import main
+
+pytest_plugins = ["tests.pytest_plugins.common", "tests.pytest_plugins.llvm"]
+
+
+@pytest.fixture(scope="module")
+def anghabench_dataset() -> AnghaBenchDataset:
+    env = gym.make("llvm-v0")
+    try:
+        ds = env.datasets["anghabench-v0"]
+    finally:
+        env.close()
+    yield ds
+
+
+def test_anghabench_size(anghabench_dataset: AnghaBenchDataset):
+    if sys.platform == "darwin":
+        assert anghabench_dataset.size == 1042908
+    else:
+        assert anghabench_dataset.size == 1042976
+
+
+@skip_on_ci
+@pytest.mark.parametrize("index", range(250))
```

Comment:
Does this mean that the test will be run for index between 0 and 249?

Reply:
Yep. `index` here is used to make this test run on a bunch of different benchmarks from the dataset, equivalent to:

```py
for uri in islice(anghabench_dataset.benchmark_uris(), 250):
    ...
```